pipelines/coordinatePipeline.py

```diff
@@ -0,0 +1,8 @@
+from genericPipeline import GenericPipeline
+
+
+class CoordinatePipeline(GenericPipeline):
```

Comment:
Why does this exist? We can have two variables of the same type that are used for different things.

Reply:
I'm not too sure about this design myself, I wanted to have an abstract class that all other pipelines could instantiate, but I took a lazy approach to it. I think I'm just going to not make generic abstract and instead make it the base pipeline and other modified pipelines can extend it